smack-extensions/src/main/resources/org.jivesoftware.smack.extensions/extensions.providers

```diff
@@ -587,6 +587,13 @@
           <elementName>mood</elementName>
           <namespace>http://jabber.org/protocol/mood</namespace>
           <className>org.jivesoftware.smackx.mood.provider.MoodProvider</className>
-        </extensionProvider>
+    </extensionProvider>
```

Comment:
Unrelated whitespace change, please revert that.

Reply:
This change was intentional.
[Master branch](https://github.com/igniterealtime/Smack/blob/master/smack-extensions/src/main/resources/org.jivesoftware.smack.extensions/extensions.providers).

Reply:
But it is unrelated to the xep118 implementation and hence should be in an extra commit.